code/datums/components/mechComp_signals.dm

```diff
@@ -280,7 +299,7 @@
 			switch(selected_config)
 				if(SET_SEND)
 					var/inp = input(user,"Please enter Signal:","Signal setting","1") as text
-					if(!in_range(parent, user) || user.stat)
+					if(!in_range(parent, user) || user.stat) //MarkNstein - check nested config setting at range
```

Comment:
This is wrong, my old comment shouldn't be here. I suspect a faulty merge-conflict resolution